tests/model_fields/test_jsonfield.py

```diff
@@ -506,11 +506,11 @@ def test_none_key(self):
     def test_none_key_exclude(self):
         obj = NullableJSONModel.objects.create(value={'j': 1})
         if connection.vendor == 'oracle':
-            # On Oracle, the query returns JSON objects and arrays that do not have a 'null' value
-            # at the specified path, including those that do not have the key.
+            # Oracle support filtering JSON objects with NULL keys, but the
```

Comment:
`support` → `supports`

Reply:
I'd like to add, shouldn't it be `'null'` instead of `NULL`?

Reply:
@laymonage No, it's a key value.